db/projects/waves/asset-waves-logo-alternative.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is big mess with Waves logos.

logo-square and logo-full is not actual, actual logo is logo-alternative, but thats some damaged version, so please:

1.  remove logo-alternative
2. set logo-full = https://svgur.com/i/8LS.svg
3. set logo-square = https://datnoid.com/coins/WAVES/waves.svg

Reply:
OK, done